src/main/java/ca/corefacility/bioinformatics/irida/util/IridaFiles.java

```diff
@@ -79,14 +86,21 @@ public static String getFileExtension(List<? extends SequencingObject> files) th
 	}
 
 	/**
-	 * Check if file exists
-	 * Checks if the file exists
+	 * Gets the file size in bytes of the file from the iridaFileStorageUtility
 	 *
 	 * @param file The path to the file
-	 * @return if file exists or not
+	 * @return file size in bytes
 	 */
-	public static boolean fileExists(Path file) {
-		return iridaFileStorageUtility.fileExists(file);
+	public static Long getFileSizeBytes(Path file) {
+		return iridaFileStorageUtility.getFileSizeBytes(file);
 	}
 
+	/**
+	 * Checks if the file exists in iridaFileStorageUtility
+	 *
+	 * @param file The path to the file
+	 * @return if file exists or not
+	 */
+	public static boolean fileExists(Path file) { return iridaFileStorageUtility.fileExists(file);}
```

Comment:
Format this to expand out the one-liner please.

Reply:
Formatted in 50d4f4fd7d2a884fc1dc8abc61771bc106536aad